cleanup externalSchedulerEnable (#382)
kubernetes-sigs/kube-scheduler-simulator: simulator/docs/integrate-your-scheduler.md

## Integrate your scheduler 

You can integrate your scheduler into the simulator.

> To add your custom scheduler plugins, refer to the following resources:
> - [debuggable-scheduler.md](./debuggable-scheduler.md#integrate-your-plugins-to-the-simulator) for basic integration steps.
> - [custom-plugin.md](./custom-plugin.md) for more detailed guidance on configuring out-of-tree plugins.

Reviewer: Ah, actually, we can remove `custom-plugin.md` as well because users can just register their custom plugins via the debuggable scheduler.

So, here can be simplified like:

```suggestion
To add your custom scheduler plugins, see [debuggable-scheduler.md](./debuggable-scheduler.md#integrate-your-plugins-to-the-simulator).
```

And, also please remove the page `custom-plugin.md`.
